Makefile

```diff
@@ -83,10 +83,21 @@ pam_radius_auth.so: src/pam_radius_auth.o src/md5.o
 #
 #  Check a distribution out of the source tree, and make a tar file.
 #
+BRANCH = $(shell git rev-parse --abbrev-ref HEAD)
+
+pam_radius-$(VERSION).tar.gz: .git
```

Comment:
GNU make doesn't handle directory dependencies properly.  :(  This should instead be `.git/HEAD`